runtime-image/Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN npm install --unsafe-perm semver
 
 # Set common env vars
 ENV NODE_ENV production
+ENV PORT 8080
```

Comment:
@dlorenc I needed to add this because in `gcr.io/google-appengine/debian8` PORT was already set to 8080, but with `gcr.io/gcp-runtimes/ubuntu_16_0_4:latest` the PORT environment variable is not set.

It seems like it might make sense to have PORT set in the `gcr.io/gcp-runtimes/ubuntu_16_0_4:latest` image.  What do you think?

Reply:
We're not going to add PORT to this ubuntu base, it was added to debian by mistake and we couldn't really drop it easily. These are general purpose base images, so the PORT should be set by the layers that need it.

Reply:
Ok that makes sense.